### PR TITLE
add dropdown menu to chart point

### PR DESCRIPTION
**Pull request by Serhiy91**

### CODE REVIEW

File: app/imports/ui/react/components/PopoverPoint.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  border-radius: 0;
+  border: none;
+  padding: 0;
+  background-color: transparent;
+  font-size: inherit;
+  top: 3px !important;
+`;
+
+const enhance = compose(
+  withState('isOpen', 'togglePopover', false),
```
Comment (Dartv) [inline app/imports/ui/react/components/PopoverPoint.js]: We have a special hoc for this `withStateToggle`

File: app/imports/ui/react/components/PopoverPoint.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  renameProps({
+    children: 'PopoverContent',
```
Comment (Dartv) [inline app/imports/ui/react/components/PopoverPoint.js]: Why not just add `popoverContent` prop?

**Comment by Serhiy91** [comment]
@Dartv Can you retest? Because It works for me https://drive.google.com/file/d/1nkwnTRGhNlaUS7gxZd4lGEUHYPUOFEuM/view

**Comment by Dartv** [comment]
Nope, doesn't work for me on chrome ubuntu 16.04

**Comment by Dartv** [comment]
👍

**Pull request merged**